geotrek/trekking/parsers.py

```diff
@@ -1588,3 +1592,11 @@ def end(self):
                 otc.save()
                 order += 1
         super().end()
+
+    @staticmethod
+    def find_url_in_description(trek):
```

Comment:
Since this method is only used in `test_parsers.py`, can you move it to the class `SchemaRandonneeParserTests`  please ? That way it also won't need to be static

Reply:
The name of the method might need to be modified so it's understandable that not any url will be retrieved.

Maybe `find_source_file_url_in_description` since this is how this url is described in `schema_randonnee`?
https://github.com/PnX-SI/schema_randonnee/blob/9a7e4a826e9af22c0a83b718a67a24b476c0c807/schema.json#L87